packages/core/stories/styles/index.scss

```
@@ -96,6 +97,11 @@ ul.sg-colors {
   }
 }
 
+// Links story
+span.ma__decorative-link  {
+	display: inline !important;
```

```suggestion
	display: inline;
```

hmm interesting, I think when I refactor the stories will resolve this. But for now I tested, the important flag is not necessary.

---

Great!